src/eosjs-serialize.ts

```diff
@@ -419,6 +419,9 @@ export class SerialBuffer { // tslint:disable-line max-classes-per-file
 
     /** Append a `symbol` */
     public pushSymbol({ name, precision }: { name: string, precision: number }) {
+        if (!/^[A-Z]{1,7}$/.test(name)) {
+            throw new Error('Expected symbol to be A-Z and 1-7 characters');
```

Comment:
This sounds like it includes the characters 1-7

Reply:
Updated. Was attempting to keep the message short.